api/preprints/views.py

```diff
@@ -197,6 +197,34 @@ def get_parser_context(self, http_request):
         res['legacy_type_allowed'] = True
         return res
 
+    def perform_update(self, serializer):
```

Comment:
this will make sure the file is there, but will it change the machine state? Are there tests you can write?